js/testing/util.ts

```diff
@@ -13,19 +13,26 @@
    limitations under the License.
  */
 
-// TODO(ry) Use unknown here for parameters types.
-// tslint:disable-next-line:no-any
-export function assertEqual(actual: any, expected: any, msg?: string) {
-  if (!msg) {
-    msg = `actual: ${actual} expected: ${expected}`;
-  }
+export function assertEqual(actual: unknown, expected: unknown, msg?: string) {
   if (!equal(actual, expected)) {
+    let actualString: string;
+    let expectedString: string;
+    try {
+      actualString = String(actual);
+      expectedString = String(expected);
```

Comment:
shouldn't you nest another try-catch here? So you can get one or the other

Reply:
Yes.  Fixed.